src/nativestore/PropertyLink.cpp

```diff
@@ -88,7 +85,29 @@ PropertyLink::PropertyLink(unsigned int blockAddress, std::string name, char* rv
  * if propertyAddress was 0 before inserting you will get head address of the link in return value
  *  else either updated link address or last appended link address will be returned
  * **/
-unsigned int PropertyLink::insert(std::string name, char* value) {
+unsigned int PropertyLink::insert(std::string name, const char* value) {
+    // TODO(thevindu-w): Temporarily commented to resolve conflict
```

Comment:
Is there any side effects of this commenting?

Reply:
No side effects. Now it has the old code before optimization.

Reply:
Ok